Sources/AudioKit/Nodes/Mixing/Mixer.swift

```diff
@@ -90,6 +90,18 @@ public class Mixer: Node, NamedNode {
         avAudioNode.disconnect(input: node.avAudioNode)
     }
 
+    /// Remove input from a mixer who's input is connected to another mixer (a split connection)
+    /// - Parameter node: Node to remove
+    /// - Parameter expectSplitConnections: Does the input to remove also have another connection (is it splitting)?
+    public func removeInput(_ node: Node, expectSplitConnections: Bool = false) {
```

Comment:
Do we need `expectSplitConnections`?

Reply:
That's what I was thinking, although whatever I did caused more of the node tests to fail. I'll keep looking at it some more though.

Reply:
Now I understand what you were saying. I tried something different before, but what I did just now seemed to work. Thanks for the advice, and let me know if there's anything else that needs to be fixed :) !